test/aws/rails/middleware/ebs_sqs_active_job_middleware_test.rb

```diff
@@ -76,6 +76,46 @@ module Rails
         expect(response[0]).to eq(500)
       end
 
+      it 'successfully invokes job when docker default gateway ip is changed' do
+        mock_rack_env = create_mock_env('192.168.176.1', 'aws-sqsd/1.1', false)
+        test_middleware = EbsSqsActiveJobMiddleware.new(mock_rack_app)
+
+        proc_net_route = <<~CONTENT
+          Iface\tDestination\tGateway\tFlags\tRefCnt\tUse\tMetric\tMask\tMTU\tWindow\tIRTT
+          eth0\t00000000\t01B0A8C0\t0003\t0\t0\t0\t00000000\t0\t0\t0
+          eth0\t00B0A8C0\t00000000\t0001\t0\t0\t0\t00F0FFFF\t0\t0\t0
+        CONTENT
+
+        allow(File).to receive(:exist?).and_call_original
```

Comment:
Are these required? Does it fail with unexpected calls without this?

Reply:
Yes.
If this line is not present, the test will fail with following error:
```shell
$ BUNDLE_GEMFILE=./gemfiles/rails-7.0.gemfile bundle exec rake
Run options: --seed 65331

# Running:

..........................................................F

Failure:
Aws::Rails::EbsSqsActiveJobMiddleware#test_0007_successfully invokes job when docker default gateway ip is changed [/home/bongole/.rbenv/versions/3.2.1/lib/ruby/gems/3.2.0/gems/rspec-mocks-3.12.3/lib/rspec/mocks/proxy.rb:217]:
#<File (class)> received :exist? with unexpected arguments
  expected: ("/proc/net/route")
       got: ("/home/bongole/.rbenv/versions/3.2.1/lib/ruby/gems/3.2.0/gems/activesupport-7.0.4.2/lib/active_support/locale/en.yml")


rails test /home/bongole/dev/aws-sdk-rails-bongole/test/aws/rails/middleware/ebs_sqs_active_job_middleware_test.rb:79

...........................................................

Finished in 2.273635s, 51.8993 runs/s, 113.4747 assertions/s.
118 runs, 258 assertions, 1 failures, 0 errors, 0 skips
rake aborted!
```